templates/reconsent-modal.php

```diff
@@ -33,7 +33,7 @@
 									<?php
 									$policy_counter = 1;
 									foreach ( $args['updated_consents'] as $consent_id => $consent ) :
-										echo '<li><button' . ( 1 === $policy_counter ? ' class="gdpr-active"' : '' ) . ' type="button" data-target="' . esc_attr( $consent_id ) . '" ' . '>' . esc_html( $consent['name'] ) . '</button></li>';
+										echo '<li><button' . ( 1 === $policy_counter ? ' class="gdpr-active"' : '' ) . ' type="button" data-target="' . esc_attr( $consent_id ) . '" ' . '>' . esc_html_e( $consent['name'] ) . '</button></li>';
```

Comment:
For this one, I think you can tell VIP to jog on with a `phpcs:ignore`

Reply:
or I think you can do `echo esc_html`

Reply:
I've run phpcs over this line locally, and to me it looks as though the original one passes phpcs. I'm guessing that this instance was updated accidentally as part of a bulk replace when addressing [this comment](https://github.com/boxuk/GDPR/pull/57#discussion_r798421163)

Reply:
That makes sense.

Hitting `Replace all`: it's risky!

Reply:
Ding ding we have a winner. I've changed it back to esc_html